Makefile

```
@@ -17,6 +17,10 @@ PROJECT := kilo
 PKG := github.com/squat/$(PROJECT)
 REGISTRY ?= index.docker.io
 IMAGE ?= squat/$(PROJECT)
+ifneq ($(REGISTRY),index.docker.io)
+    REGISTRY_REFIX := $(REGISTRY)/
+endif
+MANIFEST_IMAGE := $(REGISTRY_REFIX)$(IMAGE)
```

Small naming suggestion: for me it would be clearer if it were something like
```suggestion
FULLY_QUALIFIED_IMAGE := $(REGISTRY_REFIX)$(IMAGE)
```